airbyte-webapp/src/pages/ConnectionPage/pages/CreationFormPage/CreationFormPage.tsx

```diff
@@ -130,10 +136,10 @@ export const CreationFormPage: React.FC = () => {
       if (currentEntityStep === EntityStepsTypes.SOURCE) {
         return (
           <>
-            {type === EntityStepsTypes.CONNECTION && (
+            {/* // exp-signup-selected-source-definition */}
+            {type === EntityStepsTypes.CONNECTION && !isSourceDefinitionSelected && (
```

Comment:
I needed to add this check because otherwise it crashed.
As this flow will only happen after signup, there won't be any existing sources anyway, so I think it would be safe.
Please let me know if you feel this is not the case